resspect/scripts/fit_dataset.py

```diff
@@ -69,20 +69,20 @@ def fit_dataset(user_choices):
     features_file = user_choices.output
     ncores = user_choices.ncores
 
-    if user_choices.sim_name in ['SNPCC', 'snpcc']:
+    if user_choices.sim_name.lower() == 'snpcc':
```

Comment:
This reduces the number of combinations of upper and lower case that we need to check for.